Integrate-Kobiton.md

```diff
@@ -0,0 +1,133 @@
+# Run Kobiton Automation Test on CircleCI
+
+This document will guide you how to use CircleCI to execute automation test on Kobiton devices everytime you push a commit to GitHub.
+
+## Table of contents
+- [A. Preparation](#a-preparation)
+    - [1. Prerequisites](#1-prerequisites)
+    - [2. Sample](#2-sample)
+- [B. Setup](#b-setup)
+    - [1. Getting required parameters](#1-getting-required-parameters)
+    - [2. Configuring automation test script](#2-configuring-automation-test-script)
+- [C. Automation Test Execution](#c-automation-test-execution)
+    - [1. Executing the automation test](#1-executing-the-automation-test)
+    - [2. Fetch test session data using REST API](#2-fetch-test-session-data-using-rest-api)
+- [D. Feedback](#d-feedback)
+
+## A. Preparation
+### 1. Prerequisites
+- Kobiton account.
+    > If your don't have one, go to https://portal.kobiton.com/register to create an account.
```

Comment:
* Typo mistake: `If your don't have one`, `your` -> `you`.
* I don't think we should have quote `>` for this content.

Reply:
`Prerequisites` section just have one item, are you sure?